### PR TITLE
[Several] Cleans up curl commands

### PR DESCRIPTION
**Pull request by marciocloudflare**
Addresses PCX-6853 and follows our guidelines regarding curl commands

### CODE REVIEW

File: content/magic-firewall/how-to/collect-pcaps.md [hunk 1]
```diff
@@ -75,20 +75,20 @@ Leave `filter_v1` empty to collect all packets without any filtering.
...
+--header "X-Auth-Key: <API_KEY>" \
+--data '{
+  "filter_v1": {},
+  "time_limit": 300,
+  "packet_limit": 10000,
+  "byte_limit": 100000000,
+  "type": "full",
+  "colo": "ORD",
+  "system": "magic-transit",
+  "destination_conf": "${bucket}"
```
Comment (jason-cf) [inline content/magic-firewall/how-to/collect-pcaps.md]:
```suggestion
  "destination_conf": "${BUCKET}"
```

File: content/magic-firewall/how-to/collect-pcaps.md [hunk 2]
```diff
@@ -133,23 +133,23 @@ Leave `filter_v1` to collect all packets without any filtering.
...
-                  "destination_address": "5.6.7.8",
-                  "destination_port": 80,
-                  "protocol": 6
-          },
-          "time_limit": 300,
-          "packet_limit": 10000,
-          "type": "simple",
-          "system": "magic-transit"
-  }'
+curl https://api.cloudflare.com/client/v4/accounts/{account_id}/pcaps \
```
Comment (jason-cf) [inline content/magic-firewall/how-to/collect-pcaps.md:136]:
```suggestion
curl --request POST https://api.cloudflare.com/client/v4/accounts/{account_id}/pcaps \
```

File: content/email-security/reporting/siem-integration/splunk-integration-guide.md [hunk 3]
```diff
@@ -54,9 +54,12 @@ When Area 1 detects a phishing email, the metadata of the detection can be sent
 To test your the HTTP Event Collector, you can manually inject an event into Splunk by using the following cURL command:
 
 ```bash
-$ curl https://<HOST>:8088/services/collector/event \
--H 'Authorization: Splunk <YOUR_TOKEN>' \
--d '{"sourcetype": "<YOUR_SOURCE_TYPE", "event":"Hello, World!"}'
+curl https://{host}:8088/services/collector/event \
```
Comment (jason-cf) [inline content/email-security/reporting/siem-integration/splunk-integration-guide.md:57]:
```suggestion
curl --request POST https://{host}:8088/services/collector/event \
```

File: content/magic-firewall/how-to/enable-ids.md [hunk 4]
```diff
@@ -17,10 +17,10 @@ Follow instructions in the [Rulesets Engine Page](/ruleset-engine/basic-operatio
 Next, create a root ruleset with a single rule in the `magic_transit_ids_managed` phase by running:
 
 ```bash
-curl -X POST "https://api.cloudflare.com/client/v4/accounts/${account_id}/rulesets" \
--H 'Content-Type: application/json' \
--H 'X-Auth-Email: user@example.com' \
--H 'X-Auth-Key: 00000000000' \
+curl https://api.cloudflare.com/client/v4/accounts/{account_id}/rulesets \
```
Comment (jason-cf) [inline content/magic-firewall/how-to/enable-ids.md:20]:
```suggestion
curl --request POST https://api.cloudflare.com/client/v4/accounts/{account_id}/rulesets \
```

File: content/magic-firewall/how-to/enable-managed-rulesets.md [hunk 5]
```diff
@@ -43,10 +43,10 @@ The example below contains a request for a Managed phase Managed Kind ruleset.
 ---
 header: Example request - Create a Managed phase Managed Kind ruleset
 ---
-curl -X POST "https://api.cloudflare.com/client/v4/accounts/<ACCOUNT_ID>/rulesets"
--H 'X-Auth-Email: user@example.com' \
--H 'X-Auth-Key: 00000000000' \
--H "Content-Type:application/json" \
+curl https://api.cloudflare.com/client/v4/accounts/{account_id}/rulesets
```
Comment (jason-cf) [inline content/magic-firewall/how-to/enable-managed-rulesets.md:46]:
```suggestion
curl --request POST https://api.cloudflare.com/client/v4/accounts/{account_id}/rulesets
```

File: content/magic-firewall/how-to/pcaps-bucket-setup.md [hunk 6]
```diff
@@ -62,11 +62,11 @@ The `bucket` field should be the URI of the bucket. For Amazon S3, the `bucket`
 ---
 header: Ownership challenge request example
 ---
-curl -X POST https://api.cloudflare.com/client/v4/accounts/${account_id}/pcaps/ownership \
--H 'Content-Type: application/json' \
--H "X-Auth-Email: ${email}" \
--H "X-Auth-Key: ${auth_key}" \
--d '{
+curl https://api.cloudflare.com/client/v4/accounts/{account_id}/pcaps/ownership \
```
Comment (jason-cf) [inline content/magic-firewall/how-to/pcaps-bucket-setup.md:65]:
```suggestion
curl  --request POST https://api.cloudflare.com/client/v4/accounts/{account_id}/pcaps/ownership \
```

File: content/magic-firewall/how-to/pcaps-bucket-setup.md [hunk 7]
```diff
@@ -97,14 +97,14 @@ Validate the bucket by inserting the copied text in the `ownership_text` below:
...
-----
-curl -X POST https://api.cloudflare.com/client/v4/accounts/${account_id}/pcaps/ownership/validate \
--H 'Content-Type: application/json' \
--H "X-Auth-Email: ${email}" \
--H "X-Auth-Key: ${auth_key}" \
--d '{
-       "destination_conf": "'${bucket}'",
-       "ownership_challenge": "'${ownership_text}'"
+---
+curl https://api.cloudflare.com/client/v4/accounts/{account_id}/pcaps/ownership/validate \
```
Comment (jason-cf) [inline content/magic-firewall/how-to/pcaps-bucket-setup.md:101]:
```suggestion
curl  --request POST https://api.cloudflare.com/client/v4/accounts/{account_id}/pcaps/ownership/validate \
```

File: content/magic-firewall/how-to/pcaps-bucket-setup.md [hunk 8]
```diff
@@ -159,10 +159,10 @@ The list of buckets associated with your account displays.
 ---
 header: Bucket list request example
 ---
-curl -X GET https://api.cloudflare.com/client/v4/accounts/${account_id}/pcaps/ownership \
+curl https://api.cloudflare.com/client/v4/accounts/{account_id}/pcaps/ownership \
 -H 'Content-Type: application/json' \
--H "X-Auth-Email: ${email}" \
--H "X-Auth-Key: ${auth_key}"
+-H "X-Auth-Email: <YOUR_EMAIL>" \
+-H "X-Auth-Key: <API_KEY>"
```
Comment (jason-cf) [inline content/magic-firewall/how-to/pcaps-bucket-setup.md]:
```suggestion
--header "X-Auth-Email: <YOUR_EMAIL>" \
--header "X-Auth-Key: <API_KEY>"
```

File: content/magic-firewall/how-to/use-rules-list.md [hunk 9]
```diff
@@ -13,48 +13,48 @@ meta:
 To use this feature first [create a new IP list](/api/operations/lists-create-a-list).
 
 ```bash
-curl -X POST https://api.cloudflare.com/client/v4/accounts/${account_id}/rules/lists \
--H 'Content-Type: application/json' \
--H 'X-Auth-Email: user@example.com' \
--H 'X-Auth-Key: 00000000000' \
+curl https://api.cloudflare.com/client/v4/accounts/{account_id}/rules/lists \
```
Comment (jason-cf) [inline content/magic-firewall/how-to/use-rules-list.md:16]:
```suggestion
curl --request POST https://api.cloudflare.com/client/v4/accounts/{account_id}/rules/lists \
```

File: content/magic-firewall/how-to/use-rules-list.md [hunk 10]
```diff
@@ -13,48 +13,48 @@ meta:
...
 ```bash
-    curl -X POST https://api.cloudflare.com/client/v4/accounts/${account_id}/rules/lists/${list_id}/items \
-    -H 'Content-Type: application/json' \
-    -H 'X-Auth-Email: user@example.com' \
-    -H 'X-Auth-Key: 00000000000' \
-    --data '[
-        {"ip":"10.0.0.1"},
-        {"ip":"10.10.0.0/24"}
-    ]'
+curl POST https://api.cloudflare.com/client/v4/accounts/{account_id}/rules/lists/{list_id}/items \
```
Comment (jason-cf) [inline content/magic-firewall/how-to/use-rules-list.md:32]:
```suggestion
curl --request POST https://api.cloudflare.com/client/v4/accounts/{account_id}/rules/lists/{list_id}/items \
```

File: content/magic-firewall/how-to/use-rules-list.md [hunk 11]
```diff
@@ -13,48 +13,48 @@ meta:
...
-    --data '{
-        "action": "skip",
-        "action_parameters": {
-            "ruleset": "current"
-        },
-        "expression": "ip.src in $iplist",
-        "description": "Allowed IPs from iplist",
-        "enabled": true
-    }'
+curl https://api.cloudflare.com/client/v4/accounts/{account_id}/rulesets/{ruleset_id}/rules \
```
Comment (jason-cf) [inline content/magic-firewall/how-to/use-rules-list.md:47]:
```suggestion
curl --request POST https://api.cloudflare.com/client/v4/accounts/{account_id}/rulesets/{ruleset_id}/rules \
```

File: content/magic-transit/_partials/_update-tunnel-health-checks-frequency.md [hunk 12]
```diff
@@ -16,10 +16,12 @@ Here is an example of how you would adjust health check frequency to `low`. Note
 
 ```bash
 curl --request PUT \
```
Comment (jason-cf) [inline content/magic-transit/_partials/_update-tunnel-health-checks-frequency.md:18]: you can put the URL on this line. it's not that long.

File: content/magic-wan/third-party/aruba-edgeconnect.md [hunk 13]
```diff
@@ -180,11 +180,21 @@ For additional information on creating IPsec tunnels, refer to [API documentatio
 ---
 header: Request
 ---
-curl -X POST "https://api.cloudflare.com/client/v4/accounts/<account_id>/magic/ipsec_tunnels?validate_only=true" \
-     -H "X-Auth-Email: user@example.com" \
-     -H "X-Auth-Key: XXXXXXXXXX" \
-     -H "Content-Type: application/json" \
-     --data '{"ipsec_tunnels":[{"name":"EdgeConnect_IPSEC_1","customer_endpoint":"35.188.72.56","cloudflare_endpoint":"172.64.241.205","interface_address":"192.168.10.11/31","description":"Tunnel for EdgeConnect - GCP Central"}]}'
+curl "https://api.cloudflare.com/client/v4/accounts/<account_id>/magic/ipsec_tunnels?validate_only=true" \
+--header "X-Auth-Email: <EMAIL>" \
```
Comment (jason-cf) [inline content/magic-wan/third-party/aruba-edgeconnect.md]:
```suggestion
--header "X-Auth-Email: <YOUR_EMAIL>" \
```

File: content/magic-wan/third-party/aruba-edgeconnect.md [hunk 14]
```diff
@@ -193,11 +203,21 @@ curl -X POST "https://api.cloudflare.com/client/v4/accounts/<account_id>/magic/i
 ---
 header: Request
 ---
-curl -X POST "https://api.cloudflare.com/client/v4/accounts/<account_id>/magic/ipsec_tunnels" \
-     -H "X-Auth-Email: user@example.com" \
-     -H "X-Auth-Key: XXXXXXXXXX" \
-     -H "Content-Type: application/json" \
---data '{"ipsec_tunnels":[{"name":"EdgeConnect_IPSEC_1","customer_endpoint":"35.188.72.56","cloudflare_endpoint":"172.64.241.205","interface_address":"192.168.10.11/31","description":"Tunnel for EdgeConnect - GCP Central"}]}'
+curl https://api.cloudflare.com/client/v4/accounts/{account_id}/magic/ipsec_tunnels \
+--header "X-Auth-Email: <EMAIL>" \
```
Comment (jason-cf) [inline content/magic-wan/third-party/aruba-edgeconnect.md]:
```suggestion
--header "X-Auth-Email: <YOUR_EMAIL>" \
```

File: content/magic-wan/third-party/aruba-edgeconnect.md [hunk 15]
```diff
@@ -238,28 +258,28 @@ Use the tunnel ID from the response in Step 2. Save the pre-shared key generated
 ---
 header: Request
 ---
-curl -X POST "https://api.cloudflare.com/client/v4/accounts/<account_id>/magic/ipsec_tunnels/e70536b11daa47e09ff046fbb9800e4f/psk_generate?validate_only=true" \
-     -H "X-Auth-Email: user@example.com" \
-     -H "X-Auth-Key: XXXXXXXXXX" \
-     -H "Content-Type: application/json"
+curl --request POST "https://api.cloudflare.com/client/v4/accounts/{account_id}/magic/ipsec_tunnels/{tunnel_id}/psk_generate?validate_only=true" \
+--header "X-Auth-Email: <EMAIL>" \
```
Comment (jason-cf) [inline content/magic-wan/third-party/aruba-edgeconnect.md]:
```suggestion
--header "X-Auth-Email: <YOUR_EMAIL>" \
```

File: content/magic-wan/third-party/sonicwall.md [hunk 16]
```diff
@@ -181,10 +181,10 @@ You have to [configure Magic WAN health checks](/magic-wan/how-to/run-tunnel-hea
 
 ```bash
 curl --request PUT \
-  --url https://api.cloudflare.com/client/v4/accounts/<account_identifier>/magic/ipsec_tunnels/<tunnel_identifier> \
-  --header 'Content-Type: application/json' \
-  --header 'X-Auth-Email: <YOUR_EMAIL> ' \
-  --data '{
+https://api.cloudflare.com/client/v4/accounts/{account_id}/magic/ipsec_tunnels/{tunnel_id} \
+--header 'Content-Type: application/json' \
+--header 'X-Auth-Email: <EMAIL> ' \
```
Comment (jason-cf) [inline content/magic-wan/third-party/sonicwall.md]:
```suggestion
--header 'X-Auth-Email: <YOUR_EMAIL> ' \
--header "X-Auth-Key: <API_KEY>" \
```

File: content/magic-wan/third-party/sophos-firewall.md [hunk 17]
```diff
@@ -192,10 +192,10 @@ system gre route add net <IP_ADDRESS> tunnelname <TUNNEL_NAME>
 
 ```bash
 curl --request PUT \
-  --url https://api.cloudflare.com/client/v4/accounts/<account_identifier>/magic/ipsec_tunnels/<tunnel_identifier> \
-  --header 'Content-Type: application/json' \
-  --header 'X-Auth-Email: <YOUR_EMAIL> ' \
-  --data '{
+https://api.cloudflare.com/client/v4/accounts/{account_id}/magic/ipsec_tunnels/{tunnel_id} \
+--header 'Content-Type: application/json' \
+--header 'X-Auth-Email: <EMAIL> ' \
```
Comment (jason-cf) [inline content/magic-wan/third-party/sophos-firewall.md]:
```suggestion
--header 'X-Auth-Email: <YOUR_EMAIL> ' \
--header "X-Auth-Key: <API_KEY>" \
```

File: content/magic-wan/third-party/strongswan.md [hunk 18]
```diff
@@ -15,16 +15,16 @@ This can be set up [with the API](/api/operations/magic-i-psec-tunnels-update-i-
...
- --data '{
-   "health_check": {
-       "enabled":true,
-       "target":"172.64.240.252",
-       "type":"request",
-       "rate":"mid"
-   }
+https://api.cloudflare.com/client/v4/accounts/{account_id}/magic/ipsec_tunnels/{tunnel_id} \
+--header 'Content-Type: application/json' \
+--header 'X-Auth-Email: <EMAIL> ' \
```
Comment (jason-cf) [inline content/magic-wan/third-party/strongswan.md]:
```suggestion
--header 'X-Auth-Email: <YOUR_EMAIL> ' \
--header "X-Auth-Key: <API_KEY>" \
```

**Review by jason-cf — APPROVED** [review]

File: content/magic-transit/_partials/_update-tunnel-health-checks-frequency.md (hunk 12 above)
Comment (marciocloudflare) [reply content/magic-transit/_partials/_update-tunnel-health-checks-frequency.md:18]: The guidelines on the Wiki call for placing the URL on the line below, whenever we have PUT and PATCH requests

**Pull request merged**